Packs/CloudIncidentResponse/Playbooks/playbook-Cloud_Data_Exfiltration_Response.yml

```diff
@@ -0,0 +1,1179 @@
+    view: |-
+      {
+        "position": {
+          "x": 1260,
+          "y": 2380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: b2a335ad-2aaf-4e7a-8ccf-252137bebdd2
+    type: regular
+    task:
+      id: b2a335ad-2aaf-4e7a-8ccf-252137bebdd2
+      version: -1
+      name: Close Alert
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "30"
+    scriptarguments:
+      closeReason:
+        simple: True Positive - Handled by automation.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 2210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 9c4eb7f8-d21f-4ac7-83bc-167244ce5cb7
+    type: condition
+    task:
+      id: 9c4eb7f8-d21f-4ac7-83bc-167244ce5cb7
+      version: -1
+      name: Found malicious evidence based on enrichment data
+      description: "This step will check the following and if one of those conditions match, this alert is malicious/suspicious:\n\n- Is there access to a backup bucket? \n- Check IP prevalence - if the IP is a known IP in the company\n- Is the IP malicious?\n- Is there an enumeration alert associated with the same user?\n- Is the IP considered as a proxy?\n- Is there minimum access to this bucket? Will be determined by a threshold."
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "47"
+      Malicious:
+      - "64"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: IP.Malicious
+            iscontext: true
+          right:
+            value: {}
+        - operator: containsGeneral
+          left:
+            value:
+              simple: foundIncidents.details
+            iscontext: true
+          right:
+            value:
+              simple: enumeration
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Core.OriginalAlert.event
+                accessor: is_bucket_name_backup_storage
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Core.OriginalAlert.event
+                accessor: is_caller_ip_organization_vpn_ip_address
```

Comment:
It's one of the 'OR' conditions. Is non-VPN IP address enough to set the alert as malicious?

Reply:
its not the organization VPN, based on Eden - yes. will make it clearer